contrib/database/sql/conn.go

```diff
@@ -253,7 +253,7 @@ func (tc *TracedConn) injectComments(ctx context.Context, query string, mode tra
 	if span, ok := tracer.SpanFromContext(ctx); ok {
 		spanCtx = span.Context()
 	}
-	carrier := tracer.SQLCommentCarrier{Query: query, Mode: mode, DBServiceName: tc.cfg.serviceName}
```

Comment:
ext.TargetHost requires https://github.com/DataDog/consul-config/blob/master/datadog/us1.prod.dog/consul_config/peer-tags-config.ini be updated with `out.host`. 

Alternatives:

1. Use ext.NetworkDestinationName (network.destination.name) and add that to spans

2. Use `network.peer.address` from OpenTelemetry, add a new `ext.*` definition, and add that tag to spans

Reply:
as per internal discussion, this shouldn't be necessary, and this implementation works fine 👍